### PR TITLE
FSM Tweaks

### PR DESCRIPTION
**Pull request by liam-ilan**
### Changelist 
- Fixed bug related to apps/brakes conflict not checking secondary apps
- Dump voltages from load cells (no transfer function as of now, temporary change)
- Minor edit to internal load cells ocsc function
- Set up configuration structs for every FSM io layer
- Set up pwm configuration struct
- Nits

### Testing Done
- Passes unit tests

### CODE REVIEW

**Review by Lucien950 — CHANGES_REQUESTED** [review]
given CI I don't think the PWM will have any conflicts with the rest of the boards. So that's good I hope...

https://www.digitalocean.com/community/tutorials/arrow-operator-c-plus-plus

especially the `config`s. Remember that dereferences cost additional stack memory + cycles (very minimal).

Otherwise I am very happy to merge.

**Comment by Lucien950** [comment]
also it also looks like thruna FSM build is failing because we are sharing code? Perhaps split it off (but we are rm/rfing soon anyways)

File: firmware/quadruna/FSM/src/io/io_wheels.c [hunk 1]
```diff
@@ -1,7 +1,6 @@
 #include "io_wheels.h"
 #include <assert.h>
 #include <math.h>
-#include "main.h"
```
[the remark this answers is not in the document]
Comment (liam-ilan) [reply firmware/quadruna/FSM/src/io/io_wheels.c:4]: fixed arrow thing!

**Comment by liam-ilan** [comment]
> also it also looks like thruna FSM build is failing because we are sharing code? Perhaps split it off (but we are rm/rfing soon anyways)
Known issue that thruna FSM build fails since pwm configs are handled differently now, I can migrate thruna fsm to the new configs, though there isn't much of a use keeping it updated anymore. LMK what you want me to do :D

**Review by Lucien950 — APPROVED** [review]

**Pull request merged**